docker/cygnus-twitter/README.md

```diff
@@ -1,152 +1,3 @@
-```
-    
-You can check the running container (in a second terminal shell):
-
-```
-$ docker ps
-CONTAINER ID        IMAGE               COMMAND                CREATED              STATUS              PORTS                NAMES
-9ce0f09f5676        cygnus-twitter      "/cygnus-entrypoint.   About a minute ago   Up About a minute   5050/tcp, 8081/tcp   focused_kilby
-```
-
-You can check the IP address of the container above by doing:
-
-```
-$ docker inspect 9ce0f09f5676 | grep \"IPAddress\"
-        "IPAddress": "172.17.0.13",
-```
-    
-Once the IP address of the container is gotten, you may ask for the Cygnus version (in a second terminal shell):
-
-```
-$ $ curl "http://172.17.0.13:8081/v1/version"
-{"success":"true","version":"0.13.0_SNAPSHOT.5200773899b468930e82df4a0b34d44fd4632893"}
-```
-    
-
-You can stop the container as:
-
-```
-$ docker stop 9ce0f09f5676
-9ce0f09f5676
-$ docker ps
-CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
-```
-
-[Top](#top)
-
-###<a name="section3.2"></a>Using a specific configuration
-As seen above, the default configuation distributed with the image is tied to certain values that may not be suitable for you tests. Specifically:
-
-* It only works for storing streaming tweets in a temporal file (/tmp).
-* The logging level is `INFO`.
-* The logging appender is `console`.
-
-You may need to alter the above values with values of your own.
-
-[Top](#top)
-
-####<a name="section3.2.1"></a>Editing the docker files
-The easiest way is by editing both the `Dockerfile` and/or `agent.conf` file under `docker/cygnus-twitter` and building the cygnus-twitter image from the scratch.
-
-This gives you total control on the docker image.
-
-[Top](#top)
-
-####<a name="section3.2.2"></a>Environment variables
-Those parameters associated to an environment variable can be easily overwritten in the command line using the `-e` option. For instance, if you want to change the log4j logging level, simply run:
-
-    $ docker run -e LOG_LEVEL='DEBUG' cygnus-twitter
-
-[Top](#top)
-
-####<a name="section3.2.3"></a>Using volumes 
-Another possibility is to start a container with a volume (`-v` option) and map the entire configuraton file within the container with a local version of the file:
-
-    $ docker run -v /absolute/path/to/local/agent.conf:/opt/apache-flume/conf/agent.conf cygnus-twitter
-    
-Of course, you can combine volumes and environment variables overwritting:
-
-    $ docker run -v /absolute/path/to/local/agent.conf:/opt/apache-flume/conf/agent.conf -e LOG_LEVEL='DEBUG' cygnus-twitter
-    
-[Top](#top)
+Please, refer to [root docker documentation](../../../doc/cygnus-twitter/installation_and_administration_guide/install_with_docker.md) if you want to use a docker image for cygnus-twitter.
```

Comment:
This about root... what does it means in this context? I think something like "Please, refer to the [documentation](../../etc) if you want..."